### PR TITLE
Test logistic

### PR DESCRIPTION
**Pull request by TemiOjo25**

### CODE REVIEW

**Comment by TemiOjo25** [comment]
close #22132

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/jax/random.py [hunk 1]
```diff
@@ -381,3 +381,24 @@ def multivariate_normal(key, mean, cov, shape=None, dtype="float64", method="cho
...
+            "float16",
+            "bfloat16",
+        )
+    },
+    "jax",
+)
+def logistic(key, shape=(), dtype="float64"):
+    seed = _get_seed(key)
+
+    x = uniform(key = key, shape=shape, dtype=dtype, minval=ivy.finfo(dtype).eps)
```
Comment (umairjavaid) [inline ivy/functional/frontends/jax/random.py:400]: You need to implement the logistic function in the `ivy/functional/ivy/relevant_folder ` if the logistic isn't supported in any of the backends. You could create it as a compositional function similiar to how you have implemented it here. Frontend apis are wrappers to call the ivy implementation. For example this function should only call ivy.logistic

File: ivy_tests/test_ivy/test_frontends/test_jax/test_random.py [hunk 2]
```diff
@@ -1518,3 +1518,58 @@ def call():
...
+            dtype=dtype[0],
+        
+        )
+
+    ret = call()
+
+    if not ivy.exists(ret):
+        return
+
+    ret_np, ret_from_np = ret
```
Comment (umairjavaid) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_random.py:1570]: Why have you added it here? The value comparison of the groundtruth backend and frontend framework result is being carried out in the test_frontend_function. I don't think you need to implement it over here

**Review by umairjavaid — CHANGES_REQUESTED** [review]
Kindly add the logistic function as the composition function in the ivy API if it doesn't exist in any backends. Also no need to compare values outside the test function

File: ivy/functional/frontends/jax/random.py (hunk 1 above)
Comment (TemiOjo25) [reply ivy/functional/frontends/jax/random.py:400]: ok

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by danielmunioz** [comment]
Closing for inactivity and no longer needed.

**Pull request closed without merging**